src/optimize/base_optimizer.js

```diff
@@ -163,6 +163,19 @@ class BaseOptimizer {
         }, {})
       }
     };
+
+    // In the test env we need to add react-addons (and a few other bits) for the
+    // enzyme tests to work.
+    // https://github.com/airbnb/enzyme/blob/master/docs/guides/webpack.md
+    if (this.env.context.env === 'development') {
+      config.externals = {
```

Comment:
I don't think this is the right place for these (in our setup). I would personally prefer that they get added to the test harness https://github.com/elastic/kibana/blob/master/src/ui/public/test_harness/test_harness.js